src/main/webapp/i18n/en/student-dashboard.json

```diff
@@ -56,12 +56,18 @@
                 "sortExercises": "Sort exercises",
                 "newFirst": "Newest",
                 "oldFirst": "Oldest",
+                "previous": "Previous",
+                "current": "Current",
+                "future": "Future",
+                "noDueDate": "No due date",
+                "byDueDate": "by due date",
```

Comment:
I would assume that "by due date" means that exercises with the same due date are grouped together. I'm not sure what a better wording would be...

Reply:
May "by urgency"?

Reply:
I have added new translations within https://github.com/ls1intum/Artemis/pull/7363/commits/ba777891b491f980343b929869a3c670aa9cab76, do you think they are better?

Reply:
I like it